app/controllers/api/v1/group_order_articles_controller.rb

```diff
@@ -0,0 +1,95 @@
+class Api::V1::GroupOrderArticlesController < Api::V1::BaseController
+  include Concerns::CollectionScope
+
+  before_action :require_ordergroup
+  before_action :require_minimum_balance, only: [:create, :update] # destroy is ok
+  before_action :require_enough_apples, only: [:create, :update] # destroy is ok
+  # @todo allow decreasing amounts when minimum balance isn't met
+
+  def index
+    render_collection search_scope
+  end
+
+  def show
+    goa = scope.find(params.require(:id))
+    render_goa_with_oa(goa)
+  end
+
+  def create
+    GroupOrderArticle.transaction do
+      oa = order_articles_scope.find(create_params.require(:order_article_id))
+      go = current_ordergroup.group_orders.find_or_create_by!(order_id: oa.order_id)
+      goa = go.group_order_articles.create!(order_article: oa)
+      goa.update_quantities((create_params[:quantity] || 0).to_i, (create_params[:tolerance] || 0).to_i)
+      oa.update_results!
+      go.update_price!
+      render_goa_with_oa(goa)
+    end
+  end
+
+  def update
+    GroupOrderArticle.transaction do
+      goa = scope.includes(:group_order_article_quantities).find(params.require(:id))
+      goa.update_quantities((update_params[:quantity] || goa.quantity).to_i, (update_params[:tolerance] || goa.tolerance).to_i)
+      goa.order_article.update_results!
+      goa.group_order.update_price!
+      render_goa_with_oa(goa)
+    end
+  end
+
+  def destroy
+    GroupOrderArticle.transaction do
+      goa = scope.find(params.require(:id))
+      goa.destroy!
+      goa.order_article.update_results!
+      goa.group_order.update_price!
+      render_goa_with_oa(nil, goa.order_article)
+    end
+  end
+
+  private
+
+  def max_per_page
+    nil
+  end
+
+  def scope
+    current_ordergroup.group_order_articles.
+      preload(:order_article => :article).
```

Comment:
`order_article: :article`

Reply:
For including and preloading, I'd favour the 'old' syntax. 
`preload(:order_article, :article)` 
`preload(order_article: :article)`
`preload(:order_article => :article)`
The latter makes it most clear to me that `:article` is a 'child' of `:order_article`.

Reply:
What do you think?